src/main/data/facilities.json

```diff
@@ -165,7 +165,9 @@
   },
   {
     "code": "AIX",
-    "description": "ACCESS IT (OBS., SEE CINEDIGM)"
+    "description": "ACCESS IT",
+    "obsolete": true,
```

Comment:
Do we need `obsolete`? Couldn't a non-empty `obsoletedBy` be sufficient?

Reply:
Well there are cases where we don’t know the obsoltedBy, so those cases would have to be denoted `obsoletedBy: [] ` which to me, doesn’t obviously convey “is obsolete”. 

But I could be convinced otherwise...

Reply:
> Well there are cases where we don’t know the obsoltedBy, so those cases would have to be denoted `obsoletedBy: [] ` which to me, doesn’t obviously convey “is obsolete”.
> 
> But I could be convinced otherwise...

Agreed, we have at least one case of obsolete without a "by" - YMAGIFILM. There may be places that are completely shut down that we want to preserve for historical purposes (that we might want to filter out).

Reply:
Agree. Let’s make sure that obsoletedBy cannot be présent without obsolete also being present and true. Perhaps this can be done the JSON schéma itself.

Reply:
> Perhaps this can be done the JSON schéma itself.

Yep it can, and I tried, but hadn't *yet* gotten that constraint to work correctly. Interestingly, JSON Schema (apparently by design) isn't great with unique constraints. For example it doesn't seem capable of enforcing the unique constraint on code.

Reply:
Fixed at https://github.com/ISDCF/registries/pull/9/commits/ffa148b16965ba231cd9ba5a4d6be650290375fd